tox.ini

```diff
@@ -45,7 +45,7 @@ changedir=docs
 deps=
     -rrequirements-docs.txt
 commands=
-    sphinx-build -W -b html -d {envtmpdir}/doctrees . {envtmpdir}/html
+    sphinx-build -b html -d {envtmpdir}/doctrees . {envtmpdir}/html
```

Comment:
Removed the `-W` flag because docs generation currently fails due to warnings. I filed https://github.com/sqlalchemy-redshift/sqlalchemy-redshift/issues/239 to follow up on addressing the warnings.